### PR TITLE
Add code for the guide "Twinkly Earwarmer Headband"

### PR DESCRIPTION
**Pull request by TheKitty**
https://learn.adafruit.com/TwinklyEarwarmer/circuitpython-chill-sensor

### CODE REVIEW

File: GemmaM0_Headband/code.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import board
+import microcontroller
+import neopixel
+import adafruit_dotstar
+
+# Set up NeoPixel strand
+pixels = neopixel.NeoPixel(board.D1,  # NeoPixels on pin D1
+                           4,         # Number of Pixels
+                           brightness=0.2,   # Change from 0.0 to 1.0
+                           auto_write=True)  # Write strand immediately
```
Comment (brentru) [inline GemmaM0_Headband/code.py]: `auto_write=True` kwarg not required, the boolean kwarg is `True` by default. 

see: https://circuitpython.readthedocs.io/projects/neopixel/en/latest/api.html#neopixel.NeoPixel

File: GemmaM0_Headband/code.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# For the Gemma M0 onboard DotStar LED
+dotstar = adafruit_dotstar.DotStar(board.APA102_SCK, board.APA102_MOSI, 1)
+
+def deg_f(deg_c):  # Convert Celcius to Fahrenheit
+    return(deg_c * 9 / 5) + 32.0
+
+while True:
+    temp = deg_f(microcontroller.cpu.temperature)
+    if temp > 65.0:
```
Comment (brentru) [inline GemmaM0_Headband/code.py]: I prefer checking temp against a `CONST` like `ROOM_TEMP` defined at the top of the file

You'd be able to modify this paragraph in the guide to explain it if you want to go this optional route.
> If you need to make any adjustments, such as changing the "room temperature" cut-off that triggers the colors, or you have a different number of pixels attached to your Gemma M0, you can type them right into the code.

File: GemmaM0_Headband/code.py (hunk 1 above)
Comment (TheKitty) [reply GemmaM0_Headband/code.py]: changed

File: GemmaM0_Headband/code.py (hunk 2 above)
Comment (TheKitty) [reply GemmaM0_Headband/code.py]: changed

**Comment by TheKitty** [comment]
Thank you @brentru !

**Pull request merged**